confignode/src/main/java/org/apache/iotdb/confignode/consensus/response/DataNodeInfosResp.java

```diff
@@ -19,17 +19,20 @@
 package org.apache.iotdb.confignode.consensus.response;
 
 import org.apache.iotdb.common.rpc.thrift.TDataNodeInfo;
+import org.apache.iotdb.common.rpc.thrift.TDataNodesInfo;
 import org.apache.iotdb.common.rpc.thrift.TSStatus;
 import org.apache.iotdb.confignode.rpc.thrift.TDataNodeInfoResp;
 import org.apache.iotdb.consensus.common.DataSet;
 import org.apache.iotdb.rpc.TSStatusCode;
 
+import java.util.List;
 import java.util.Map;
 
 public class DataNodeInfosResp implements DataSet {
```

Comment:
The DataNodeInfosResp can be removed since you don't need consensus read

Reply:
this class is useful